### ISSUE TITLE
[Python] elastic-apm 6.8.0 : no more SQL informations in the timeline

### ISSUE THREAD

**Issue by chichi13** [elastic/apm-agent-python#1482]
**Describe the bug**: ...

**To Reproduce**

1. Python project with elastic-apm 6.8.0
2. Python project with databases/queries...

**Environment (please complete the following information)**
- OS: Docker / Python 3.10.2 / Debian bullseye
- Python version: 3.10.2
- Framework and version [e.g. Django 2.1]: FastAPI 0.75.0 / SQLAlchemy 1.4.32 
- APM Server version: 7.16.2
- Agent version: elastic-apm 6.8.0

**Additional context**

- Agent config options 
  <details>
    <summary>Click to expand</summary>

    ```
    Default Elastic Google Cloud config
    ```
  </details>
- `requirements.txt`:
  <details>
    <summary>Click to expand</summary>

    ```
    fastapi==0.75.0
    uvicorn==0.17.5
    pydantic[dotenv]==1.9.0
    requests==2.27.1
    alembic==1.7.6
    PyMySQL==1.0.2
    SQLAlchemy==1.4.32
    python-slugify==6.1.1
    elastic-apm==6.8.0
    cryptography==36.0.1
    pycountry==22.3.5
    python-jose==3.3.0
    ```
  </details>

With the upgrade of elastic-apm from 6.7.2 to 6.8.0 I've lost SQL informations in the timeline : 

![imagen](https://user-images.githubusercontent.com/25102234/157474218-152b3c35-43d0-48fc-bc7a-8f62c571327b.png)

When I downgrade elastic-apm to 6.7.2 everything is working well : 

![imagen](https://user-images.githubusercontent.com/25102234/157474425-21ab501b-04c9-4481-9d8f-5bf9a4460cf3.png)

Could you please confirm this bug ?

**Comment by chichi13** [comment]
Glad I'm not the only one.

I didn't mention it, but there is strictly no change in my APM agent/server too. Just upgrade/downgrade of the package in my requirements.txt

**Comment by basepi** [comment]
Definitely seems like a regression, perhaps related to dropping fast exit spans. Can someone try setting [`ELASTIC_APM_EXIT_SPAN_MIN_DURATION=0ms`](https://www.elastic.co/guide/en/apm/agent/python/current/configuration.html#config-exit-span-min-duration)? Curious if the spans are being dropped due to this setting.

I'm investigating.

**Comment by basepi** [comment]
This is definitely the new "dropping fast exit spans" feature. I've got a fix in the works (#1483). Apologies for the regression!

**Comment by chichi13** [comment]
Thanks for the quick fix ! I'll downgrade until the new version
